sklearn/decomposition/tests/test_sparse_pca.py

```diff
@@ -70,6 +71,10 @@ def test_fit_transform():
     spca_lasso.fit(Y)
     assert_array_almost_equal(spca_lasso.components_, spca_lars.components_)
 
+    # Test calling tranform with deprecated ridge_alpha parameter
```

Comment:
*transform

Here is fine. It's frustrating that this parameter was not already tested.